reference/edit-window/metadata/distribution-panels/onlineOption.md

```diff
@@ -0,0 +1,23 @@
+### Online Transfer Option 
+
+![Online Transfer Option Edit Window](/assets/reference/edit-objects/metadata/distribution/OnlineTransferOption.PNG)
```

Comment:
This should be moved to the [include file](https://github.com/adiwg/mdEditor-doc/blob/master/reference/include-objects/onlineResource-obj.md) and that file should be included here.

Reply:
I updated this file with the following:

" {% include "../../../include-objects/onlineTransferOption.md" %} "

I tested this locally, however; I would like a second opinion on its use.

Reply:
Before I resolve this conversation, I want to verify if you spot any issue with the change.

Reply:
It looks like the "Online Option" link on the "[Transfer Option](https://ironrico.github.io/mdEditor-doc/reference/edit-window/metadata/distribution-panels/transferOption.html)" page is pointing to https://ironrico.github.io/mdEditor-doc/reference/edit-window/metadata/distribution-panels/onlineTransferOption.md (the current page).

The table of contents "Online Options" points to: https://ironrico.github.io/mdEditor-doc/reference/include-objects/onlineTransferOption.html, which does display the correct information.